docs/source/_includes/available_permission_types.rst

```diff
@@ -3,53 +3,45 @@
-| action_create    | Ability to create a new action.                                            |
-+------------------+----------------------------------------------------------------------------+
-| action_modify    | Ability to modify (update) an existing action.                             |
-+------------------+----------------------------------------------------------------------------+
-| action_delete    | Ability to delete an existing action.                                      |
-+------------------+----------------------------------------------------------------------------+
-| action_execute   | Ability to execute (run) an action.                                        |
-+------------------+----------------------------------------------------------------------------+
-| action_all       | Ability to perform all the supported operations on a particular action.    |
-+------------------+----------------------------------------------------------------------------+
-| rule_view        | Ability to view a rule.                                                    |
-+------------------+----------------------------------------------------------------------------+
-| rule_create      | Ability to create a new rule.                                              |
-+------------------+----------------------------------------------------------------------------+
-| rule_modify      | Ability to modify (update) an existing rule.                               |
-+------------------+----------------------------------------------------------------------------+
-| rule_delete      | Ability to delete an existing rule.                                        |
-+------------------+----------------------------------------------------------------------------+
-| rule_all         | Ability to perform all the supported operations on a particular rule.      |
-+------------------+----------------------------------------------------------------------------+
-| execution_view   | Ability to view an execution.                                              |
-+------------------+----------------------------------------------------------------------------+
-| execution_re_run | Ability to create a new action.                                            |
-+------------------+----------------------------------------------------------------------------+
-| execution_stop   | Ability to stop (cancel) a running execution.                              |
-+------------------+----------------------------------------------------------------------------+
-| execution_all    | Ability to perform all the supported operations on a particular execution. |
-+------------------+----------------------------------------------------------------------------+
++-----------------+-------------------------------------------------------------------------+
+| Permission name | Description                                                             |
```

Comment:
Nope. Accidentally added this file. It's a result of running "make all".  Annoying...